packages/opensrp-plans/src/containers/PlanAssignment/index.tsx

```diff
@@ -111,7 +111,9 @@ const PlanAssignment = (props: PlanAssignmentTypes) => {
       <div className="plan-activities_section">
         {showAssignmentTable ? <ConnectedAssignmentTable {...assignmentTableProps} /> : null}
         {showActivateMission ? <ActivateMissionCard {...activateMissionProps} /> : null}
-        {showMissionData ? <MissionData baseURL={baseURL} plan={plan} /> : null}
+        {showMissionData && plan.status !== PlanStatus.DRAFT ? (
```

Comment:
I think this should also not show for retired plans,  I think we can also move the check inside the component